src/components/common/DataTabel/Tabel.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+
+import React from "react";
+import {
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+import { Member } from "@/types";
+import useGroupedData from "@/hooks/use-group-data";
+
+type DataTableProps = {
+  data: Member[];
+  columns: any;
```

Comment:
`any` isn't a great way to handle things. We should learn to type this.

Reply:
Fixed